connect_four/board.rb

```diff
@@ -0,0 +1,130 @@
+class Board
+  attr_accessor :board
+
+  def initialize
+    @board = Array.new(7) {Array.new(0)} #7 empty columns
+  end
+
+  # Graphic methods
+  def render
+    puts "\n   1 2 3 4 5 6 7"
+    5.downto 0 do |row|
+      print "  |"
+      0.upto 6 do |column|
+        if board[column][row].nil?
```

Comment:
I honestly didn't know I could just do .nil? to check if nil was in there. To see if a column was full I used .all? ... I may have used a !=nil or ==nil